be/src/Project/Services/Catalog/Catalog.Api/v1/Categories/Controllers/CategoriesController.cs

```diff
@@ -167,8 +167,7 @@ public async Task<IActionResult> Save(CategoryRequestModel request)
                     Id = request.Id,
                     Files = request.Files,
                     Name = request.Name,
-                    Schema = request.Schema,
-                    UiSchema = request.UiSchema,
+                    Schemas = request.Schemas,
```

Comment:
**suggestion:** Change to:

![obraz](https://github.com/dawiddworak88/Giuru/assets/93117743/8c90521c-4e62-4044-b65c-7986008af215)